crates/accelerate/src/sparse_observable.rs

```diff
@@ -2499,11 +2499,11 @@ impl PySparseObservable {
             // we reverse the order of bits and indices so the Pauli string comes out in
             // "reading order", consistent with how one would write the label in
             // SparseObservable.from_list or .from_label
-            for bit in view.bit_terms.iter().rev() {
+            for bit in view.bit_terms.iter() {
                 pauli_string.push_str(bit.py_label());
             }
             let py_string = PyString::new(py, &pauli_string).unbind();
-            let py_indices = PyList::new(py, view.indices.iter().rev())?.unbind();
+            let py_indices = PyList::new(py, view.indices.iter())?.unbind();
```

Comment:
Didn't we deliberately reverse these, according to the comment above?

Reply:
Whoops no -- on top of catching debugging prints, we now need something to catch debugging-undos 😂 Fixed in 3a5a5ad28